scripts/pre_push_hook.py

```diff
@@ -223,13 +224,15 @@ def _get_refs():
 
 
 def _start_linter(files):
+    """Starts the lint checks and returns the returncode of the task."""
     script = os.path.join(SCRIPTS_DIR, LINTER_SCRIPT)
     task = subprocess.Popen([PYTHON_CMD, script, LINTER_FILE_FLAG] + files)
     task.communicate()
     return task.returncode
 
 
 def _start_sh_script(scriptname):
+    """Runs the `start.sh` script and returns the returncode of the task."""
```

Comment:
Do we need to use single quotes instead of `?

Reply:
Also does this need args & returns?

Reply:
Done